### PR TITLE
Updated design for date time stacked

### PR DESCRIPTION
**Pull request by evalouderback**
This PR includes updates to Date Stacked (to add the bold variant), Date Time Stacked, and Section Separator (to get rid of that spacing on the react version for Date Time Stacked). 

I think maybe the Section Separator kit needs to be updated, because as you can see on Date Time Stacked, if you have the year displaying, the time is not center aligned. If you switch the Flex kit wrapping it to use `align-items: center` instead of `align-items: stretch`, the Section Separator for some reason disappears. This behavior also happens if you switch it to `align-items: center` on the kit itself.

#### Screens

**Date Stacked:**
Screen Shot 2020-12-16 at 11 56 06 AM
Screen Shot 2020-12-16 at 11 56 19 AM

**Date Time Stacked:**
Screen Shot 2020-12-16 at 11 55 29 AM

**Section Separator (no visual changes):**
Screen Shot 2020-12-16 at 11 56 48 AM

#### Breaking Changes

No
#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/NUXE-58)
#### How to test this

[INSERT TESTING DETAILS]

#### Checklist:

- [ ] **LABELS** Add a label: `enhancement`, `bug`, `improvement`, `new kit`, `deprecated`, or `breaking`. See [Changelog & Labels](https://github.com/powerhome/playbook/wiki/Changelog-&-Labels) for details.
- [ ] **URGENCY** Please select a release milestone
- [ ] **DEPLOY** Please add the `Milano` label when you are ready for a review.
- [ ] **SCREENSHOT** Please add a screen shot or two.
- [ ] **SPECS** Please cover your changes with specs.
- [ ] **READ DOCS** Please make sure you have read and understand the [Playbook Release Process](https://github.com/powerhome/playbook/wiki/Playbook-Releases)

*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*

### CODE REVIEW

File: playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.scss [hunk 1]
```diff
@@ -1,6 +1,15 @@
-@import "../pb_title/title";
-@import "../pb_body/body";
+[class^=pb_date_time_stacked] {
+
+  [class^=padding_month] {
+    padding-bottom: 10px !important;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.scss:4]:
```suggestion
    padding-bottom: ($space_xs + 2) !important;
```

Let's try and use variables even if they are slightly custom

File: playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.scss [hunk 2]
```diff
@@ -1,6 +1,15 @@
...
+
+  [class^=padding_day] {
+    padding-bottom: 10px !important;
+  }
 
-.pb_date_time_stacked_kit {
-  padding-left: $space-xs;
 }
+.date-time-padding {
+  padding-right: 3px;
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_date_time_stacked/_date_time_stacked.scss:13]:
```suggestion
  padding-right: ($space_xs / 2);
```

File: playbook/app/pb_kits/playbook/pb_date_time_stacked/docs/_date_time_stacked_dark.jsx [hunk 3]
```diff
@@ -7,6 +7,11 @@ const DateTimeStackedDark = () => (
         dark
         date={new Date()}
     />
+    <br />
+    <DateTimeStacked
+        dark
+        date={new Date('20 Mar 2018')}
```
Comment (jasoncypret) [inline playbook/app/pb_kits/playbook/pb_date_time_stacked/docs/_date_time_stacked_dark.jsx:13]: @coleerikson Is this date correct?

**Review by jasoncypret — CHANGES_REQUESTED** [review]
A couple of nits

**Comment by jasoncypret** [comment]
I'm not sure the dates are correct. For example, the top example is 1 hour off. The second uses eastern but is at 1a. React example:

image

**Review by jasoncypret — CHANGES_REQUESTED** [review]
Remove the dark example. No longer needed after dark mode

image

**Review by jasoncypret — CHANGES_REQUESTED** [review]
image

Time is not setup correctly

**Review by jasoncypret — CHANGES_REQUESTED** [review]
image

React dark mode not working. Follow how the docs are setup for other react kits. See the `{...props}` things...

**Comment by thestephenmarshall** [comment]
This one appears to still need some work. Any updates in flight @evalouderback?

**Comment by thestephenmarshall** [comment]
Passing this one back to you @evalouderback

**Pull request closed without merging**